content/sensu-go/6.8/operations/deploy-sensu/secure-postgres.md

```diff
@@ -0,0 +1,374 @@
+`sensu.csr`      | Certificate signing request for the Sensu backend certificate. Not used. |
+
+Now that you have the required certificates and keys, you can configure Sensu to use certificate authentication with PostgreSQL.
+
+## Configure Sensu to use certificate authentication with PostgreSQL
+
+{{% notice note %}}
+**NOTE**: The Sensu backend uses the libpq library to make connections to PostgreSQL.
+This library [supports a number of environment variables](https://www.postgresql.org/docs/current/libpq-envars.html) that can be injected into the PostgreSQL data source name (DSN) and are loaded at runtime using the system's environment variable file.
+{{% /notice %}}
+
+Working from your Sensu backend, follow these steps to configure Sensu to use certificate authentication with PostgreSQL:
+
+1. Define the environment variables that tell the Sensu backend to use a certificate to authenticate to PostgreSQL:
+
+   {{< language-toggle >}}
+
+{{< code "RHEL/CentOS" >}}
+echo 'PGUSER=sensu
+PGSSLMODE="verify-full"
+PGSSLCERT="/etc/sensu/tls/sensu.pem"
+PGSSLKEY="/etc/sensu/tls/sensu-key.pem"
+PGSSLROOTCERT="/etc/sensu/tls/ca.pem"' | sudo tee /etc/sysconfig/sensu-backend
+{{< /code >}}
+
+{{< code "Ubuntu/Debian" >}}
+echo 'PGUSER=sensu
+PGSSLMODE="verify-full"
+PGSSLCERT="/etc/sensu/tls/sensu.pem"
+PGSSLKEY="/etc/sensu/tls/sensu-key.pem"
+PGSSLROOTCERT="/etc/sensu/tls/ca.pem"' | sudo tee /etc/default/sensu-backend
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+2. Adjust the Sensu datastore connection with sensuctl:
+
+   {{< code shell >}}
+
+echo 'type: PostgresConfig
+api_version: store/v1
+metadata:
+  name: sensu_postgres
+spec:
+  dsn: "postgresql://postgres.example.com:5432/sensu_events"
+  pool_size: 20
+  strict: false' | sudo tee postgresconfig.yml
+
+sensuctl create -f postgresconfig.yml
+
+{{< /code >}}
+
+   {{% notice note %}}
+**NOTE**: Setting `strict: false` in the configuration helps ensure that the Sensu backend will remain active and able to process events even in case of a configuration mistake.
+{{% /notice %}}
+
+3. Confirm that the connection to your PostgreSQL instance is healthy:
+
+   {{< code shell >}}
+curl http://localhost:8080/health
+{{< /code >}}
+
+   The response should be similar to this example, with `true` values for both `Active` and `Healthy`:
+
+   {{< code text >}}
+{
+  "Alarms": null,
+  "ClusterHealth": [
+    {
+      "MemberID": 13217573501179607000,
+      "MemberIDHex": "b76e4111d26d35e2",
+      "Name": "sensu.example.com",
+      "Err": "",
+      "Healthy": true
+    }
+  ],
+  "Header": {
+    "cluster_id": 11959078708079102000,
+    "member_id": 6370351775894128000,
+    "raft_term": 4242
+  },
+  "PostgresHealth": [
+    {
+      "Name": "sensu_postgres",
+      "Active": true,
+      "Healthy": true
+    }
+  ]
+}
+{{< /code >}}
+
+Now that you've confirmed that the Sensu backend can connect to your PostgreSQL instance, you can configure PostgreSQL to use TLS.
```

Comment:
Should there be another step for restarting the backend at the end of this section? In the first draft, there was a line...

```
We won't restart our backend to load those environment variables just yet. There are still a few steps left to ensure that we don't inadvertently take down our backend
```

...but it didn't include an explicit step to do this